ci_framework/roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -56,23 +65,51 @@
       --ssh-inject {{ _admin_user }}:file:{{ ansible_user_dir }}/.ssh/authorized_keys
       | tee -a {{ item.value.image_local_dir }}/{{ item.value.disk_file_name }}.log
     creates: "{{ item.value.image_local_dir }}/{{ item.value.disk_file_name }}.log"
-  loop: "{{ cifmw_libvirt_manager_configuration.vms | dict2items }}"
+  loop: "{{ _layout.vms | dict2items }}"
   loop_control:
     label: "{{ item.key }}"
 
+- name: Create temporary ssh keypair
```

Comment:
Should we explicetly use `ansible_user_dir` as the base path of the SSH key?

Reply:
I had so many issues with the `ansible_user_dir` I abandoned when it came to that part. With the delegation and all, it was "slightly" messy in the end, so I ended up with relative path to the execution location, which happens to be in the user dir on the target host. Bad, good, no idea, it works... :]